gm4_chairs/data/gm4_chairs/functions/sit_counter.mcfunction

```diff
@@ -1,5 +1,7 @@
-#run from advancement sitting_in_chair
+# Increase score for time sitting in a chair
 # @s = player sitting on the chair
+# at @s
+# run from advancement sitting_in_chair
 
 advancement revoke @s only gm4_chairs:sitting_in_chair
 
```

Comment:
Using an advancement for this is actually quite bad for performance. Now that you have a predicate anyways you could just check the predicate on a _very_ slow clock.

Reply:
Would you suggest a separate function that's always running on a slow clock? (like every 10-30 seconds)

I was thinking of having this clock only run while a player is sitting on a chair, started by a player sitting on it (1.18 advancement trigger), but disadvantages of this is that relogging would stop this clock and not restart when logging back in